landlab/components/radiation/radiation.py

```diff
@@ -130,10 +184,13 @@ def __init__(
         latitude=34.0,
         albedo=0.2,
         solar_constant=1366.67,
-        clearsky_turbidity=2.0,
-        opt_airmass=0.0,
+        clearsky_turbidity=_default_assertion_val,
+        opt_airmass=_default_assertion_val,
```

Comment:
`_default_assertion_val `removed, replaced with `None`, same functionality achieved. Because turbidity could be 0 when cloud cover is negligible, 0.0 can't be a default value since the default values for turbidity and airmass are meant to specify that the airmass and turbidity should be ignored in calculations, whereas a value of 0.0 **could be used intentionally** for formulas using those two variables.